broadinstitute/orsp-pub
# BTRX-808 ORSP: Read Only Admin can create a DUL Link

## Addresses
https://broadinstitute.atlassian.net/browse/BTRX-808

## Changes
If user has read only role, should not be didplayed data use letter section
## Testing
If user has read only role, should not be didplayed data use letter section.
1) Login with a user that has read only access. Go to Consent Group - > Documents. Data Use Letter section shouldn't be displayed.
2) Login with a user that has not read only access, Data Use Letter section should be displayed.

---
